app/scripts/lib/fxa-client.js

```diff
@@ -112,6 +112,16 @@ define(function (require, exports, module) {
           });
     },
 
+    checkAccountByEmailExists: function (email) {
+      return this._getClient()
+        .then(function (client) {
+          return client.accountStatus({email:email})
```

Comment:
No worries. I will check to see if there is a eslint configuration for this. I keep missing it :(